Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Merchant.hs

```diff
@@ -94,7 +94,9 @@ instance FromTType' BeamM.Merchant Merchant where
             aadhaarKeyExpiryTime = aadhaarKeyExpiryTime,
             mediaFileSizeUpperLimit = mediaFileSizeUpperLimit,
             mediaFileUrlPattern = mediaFileUrlPattern,
-            trackingShortUrlPattern = trackingShortUrlPattern
+            trackingShortUrlPattern = trackingShortUrlPattern,
+            editPickupDistanceThreshold = fromMaybe 100 editPickupDistanceThreshold,
```

Comment:
Why to have it as Maybe value, it could be non Maybe value.

Reply:
Yes I kept it non maybe only but due to Drainer we need to keep new columns as maybe